src/common/CONSTANTS.js

```diff
@@ -40,6 +40,49 @@ const ICON_FOR_TYPE = new Map([
     ['other', 'movies'],
 ]);
 
+const EXTERNAL_PLAYERS = [
+    {
+        label: 'EXTERNAL_PLAYER_DISABLED',
+        value: null,
+        platforms: ['ios', 'android', 'windows', 'linux', 'macos'],
+    },
+    {
+        label: 'EXTERNAL_PLAYER_ALLOW_CHOOSING',
+        value: 'choose',
+        platforms: ['android'],
+    },
+    {
+        label: 'VLC',
+        value: 'vlc',
+        platforms: ['ios', 'android'],
```

Comment:
```suggestion
        platforms: ['ios', 'android', 'macos', 'windows', 'linux'],
```

Reply:
We only support VLC for ios and android for this feature

Reply:
As noted by tim, we indeed do not support them.
Also cross-checked with core:
https://github.com/Stremio/stremio-core/blob/bcd037874e1e73990a9688b06adbd551bf59709b/src/deep_links/mod.rs#L85-L91